executor/executor_test.go

```diff
@@ -6290,7 +6290,17 @@ func (s *testSuite) TestCollectDMLRuntimeStats(c *C) {
 
 	tk.MustExec("begin pessimistic")
 	tk.MustExec("insert ignore into t1 values (9,9)")
-	c.Assert(getRootStats(), Matches, "time:.*, loops:.*, BatchGet:{num_rpc:.*, total_time:.*}, lock_keys: {time:.*, region:.*, keys:.*, lock_rpc:.*, rpc_count:.*}")
+	c.Assert(getRootStats(), Matches, "time:.*, loops:.*, prepare:.*, check_insert:{total_time:.*, mem_check_insert:.*, rpc:{time:.*, BatchGet:{num_rpc:.*, total_time:.*}}}, lock_keys: {time:.*, region:.*, keys:.*, lock_rpc:.*, rpc_count:.*}")
+	tk.MustExec("rollback")
+
+	tk.MustExec("begin pessimistic")
+	tk.MustExec("insert into t1 values (10,10) on duplicate key update a=a+1")
+	c.Assert(getRootStats(), Matches, "time:.*, loops:.*, prepare:.*, check_insert:{total_time:.*, mem_check_insert:.*, rpc:{time:.*, BatchGet:{num_rpc:.*, total_time:.*}.*")
+	tk.MustExec("rollback")
+
+	tk.MustExec("begin pessimistic")
+	tk.MustExec("insert into t1 values (1,2)")
+	c.Assert(getRootStats(), Matches, "time:.*, loops:.*, prepare:.*, check_insert:{total_time:.*, mem_insert:.*}.*")
```

Comment:
```sql
 Insert_1        root    0       N/A             0       time:3.034751ms, loops:1, prepare:3.009321ms, check_insert:{total_time:25.43µs, mem_insert:25.43µs},
```
For sql `insert into t1 values (1,2)`, it doesn't have `check`, and the `total_time` value is same with mem_insert.

How about:

```sql
 Insert_1        root    0       N/A             0       time:3.034751ms, loops:1, prepare:3.009321ms, insert:25.43µs
```